sdks/java/io/amazon-web-services2/src/main/java/org/apache/beam/sdk/io/aws2/kinesis/KinesisTransformRegistrar.java

```diff
@@ -64,6 +64,7 @@ private abstract static class CrossLanguageConfiguration {
     String region;
     @Nullable String serviceEndpoint;
     boolean verifyCertificate;
+    boolean aggregationEnabled;
```

Comment:
This seems helpful - if we're going to do this, could we also expose the properties here? https://beam.apache.org/releases/javadoc/current/org/apache/beam/sdk/io/aws2/kinesis/KinesisIO.RecordAggregation.Builder.html

e.g.

```
maxAggregationBufferedTime
maxAggregationBytes
aggregationShardRefreshInterval
```